### PR TITLE
Jailbreaking Blog

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: _posts/2020-05-26-jailbreaking-for-ios-developers.md [hunk 1]
```diff
@@ -40,94 +40,94 @@ Jailbreaking has a bad taste because it can be used to pirate apps. But there ar
...
 
-# Adding Repositories to Cydia
+## Adding Repositories to Cydia
 
-[Cydia](https://cydia-app.com/) is the oldest and most common alternative App Store for iOS. It's automatically installed for most jailbreaks. It is a convenient UI for the apt-get packager that it comes with. In the earlier days you could also buy apps through Cydia, nowadays most apps are sold via 3rd-party repositories. They offer free and paid apps (via PayPal or Credit Card) and can be easily added to Cydia.
+[Cydia](https://cydia-app.com/) is the oldest and most common alternative App Store for iOS. It’s automatically installed for most jailbreaks, and it has a convenient UI for the apt-get packager that it comes with. In the earlier days, you could also buy apps through Cydia, but nowadays most apps are sold via third-party repositories. These offer free and paid apps (via PayPal or credit card) and can be easily added to Cydia. 
 
 - [Packix](https://repo.packix.com/)
```
Comment (deutschbitte) [inline _posts/2020-05-26-jailbreaking-for-ios-developers.md:67]: Are these examples of third-party repositories? Are these the ones you recommend or think are the best?

File: _posts/2020-05-26-jailbreaking-for-ios-developers.md [hunk 2]
```diff
@@ -40,94 +40,94 @@ Jailbreaking has a bad taste because it can be used to pirate apps. But there ar
...
 
-[FLEX](https://github.com/Flipboard/FLEX) is an open-source in-app debugging and exploration tool for iOS by [@NSExceptional](https://twitter.com/NSExceptional). It's amazing what you can do with it. Want the weather background as home screen background? No problem.
+[FLEX](https://github.com/Flipboard/FLEX) is an open source in-app debugging and exploration tool for iOS by [@NSExceptional](https://twitter.com/NSExceptional). It’s amazing what you can do with it. Want the weather background as yout home screen background? No problem.
 
 {% twitter https://twitter.com/nsexceptional/status/1250353513923674114 %}
 
-To Install, download [FLEXing](http://cydia.saurik.com/package/com.pantsthief.flexing/), reboot and then tap on the status bar to load FLEX. You can browse the classes and inspect the view hierarchy with a 3D debugger, similar to [Reveal](https://revealapp.com/). Here's Spotify:
+To install, download [FLEXing](http://cydia.saurik.com/package/com.pantsthief.flexing/), reboot your device, and then tap on the status bar to load FLEX. You can browse the classes and inspect the view hierarchy with a 3D debugger, similar to [Reveal](https://revealapp.com/). Here’s Spotify:
```
Comment (deutschbitte) [inline _posts/2020-05-26-jailbreaking-for-ios-developers.md:107]: Is Reveal a 3D debugger or does it just let you do the same/similar things as a debugger?

"You can browse the classes and inspect the view hierarchy with a 3D debugger, similar to [Reveal](https://revealapp.com/)."

**Comment by deutschbitte** [comment]
Is this list meant to be a list of reasons for jailbreaking?

- Security Research (MITM via [SSLKillSwitch](https://github.com/nabla-c0d3/ssl-kill-switch2), [dumpdecrypted](https://github.com/stefanesser/dumpdecrypted) to decrypt/disassemble apps)
- Enhancing iOS with tweaks (e.g. [native picture-in-picture](https://repo.packix.com/package/codes.rambo.ipadify/), [tabs in Safari](https://repo.twickd.com/package/com.twickd.minazuki.safari-electro-2), a [call-bar](https://www.idownloadblog.com/2019/03/06/callbar-xs-brings-everyones-favorite-phone-call-centric-jailbreak-tweak-to-ios-12/), [better notifications](https://www.idownloadblog.com/2019/02/25/notifica/), [better shortcuts](http://cydia.saurik.com/package/com.ethanrdoesmc.truecuts/))
- Enhancing other apps (e.g. adding a [Tomatometer score to Netflix](https://repo.packix.com/package/org.packix.flixenhancer/) or disabling ads for YouTube [not linking, as this circumvents something that Google charges money for])
- Installing apps that are not allowed on the App Store because they could be used for copyright violations ([Game Console Emulators](https://tweak-box.com/delta/)) or are only legal in certain countries ([Automatic Call Recording](http://ioscallrecorder.com/))
- Widgets on the Lock/Home Screen. (e.g. [Xen HTML](https://xenpublic.incendo.ws/))
- Theming via [SnowBoard](https://repo.packix.com/package/com.spark.snowboard/) ([Viola](https://repo.packix.com/package/com.bousrih.viola/) is what you see on the screenshot above, or [Mojito](https://repo.packix.com/package/eu.bednarz.eyeris/))
- Convenience for developers ([ssh into the device](https://twitter.com/develobile/status/1264656302195818497?s=21), [inspect view hierarchy](https://github.com/Flipboard/FLEX), a [proper file browser](http://cydia.saurik.com/package/com.tigisoftware.filza/))
- Accessing Hardware features that are otherwise inaccessible (e.g. [NFC Writer](http://cydia.saurik.com/package/net.limneos.nfcwriterx/))
- [Location spoofing](https://www.reddit.com/r/jailbreak/comments/dzdzgg/tutorial_nepetas_relocate_guide_on_1322/)
- Working around tethering restrictions ([TetherMe](http://cydia.saurik.com/package/net.tetherme.tetherme8/))
- Improving accessibility (e.g. [blind folks](https://twitter.com/devinprater/status/1264962317609046017), [special needs](https://twitter.com/GadgetGal_/status/1264962311229333504?s=20))

File: _posts/2020-05-26-jailbreaking-for-ios-developers.md (hunk 1 above)
Comment (steipete) [reply _posts/2020-05-26-jailbreaking-for-ios-developers.md:67]: Exactly! I recommend these, but there's many more.

File: _posts/2020-05-26-jailbreaking-for-ios-developers.md (hunk 2 above)
Comment (steipete) [reply _posts/2020-05-26-jailbreaking-for-ios-developers.md:107]: Reveal is a 3D debugger, but we use a different 3D debugger here (FLEX)

**Comment by steipete** [comment]
>Is this list meant to be a list of reasons for jailbreaking?

absolutely!

**Pull request merged**